# ${{\color{Black}\Huge{\textsf{Routine Codes and Pipelines}}}}\$   <br />
 ${{\color{Black}\normalsize{\textsf{Epigenetics, 3D Genome, and Functional Annotation}}}}\$   <br />
 
# Compilation of codes and pipelines used routinely in the lab <br />

# ${{\color{Green}\Huge{\textsf{Overview}}}}\$   <br />
This repository is a curated collection of computational scripts and pipelines regularly utilized in the Hawkins Lab for genome-wide studies. It provides essential tools for analyzing histone modifications, DNA methylation, 3D genome organization, and the annotation of coding and non-coding RNAs, reflecting the lab's commitment to advancing gene regulation and functional genomics research.  <br />
- **Applications**  <br />
- Functional annotation of genomic regions and chromatin states. <br />
- Characterization of gene regulatory landscapes in various cellular contexts. <br />
- 3D genome structure analysis for insights into chromatin organization and its role in gene expression. <br />

:large_orange_diamond: **Contents**:  <br />

* bcl2fastq demultiplexing NGS libraries (convert  *.bcl to *.fastq files)  <br />
:link: https://github.com/hawkinslab226/Routine_pipelines/blob/main/demultiplex_libraries_hawkinslab.sh 

### Demultiplexing NGS libraries Best Practices <br />
 - **create a sample ID label to sample sheet files** Ex: *library.name-Tissue.or.Cells-animal.ID-assay-run.date* <br />
 
 -  **$${\color{purple}T01-muscle-01-ATAC240101}$$**  <br />

 * #### Normalization raw counts  <br />



